### PR TITLE
device_support(Iluminize): add `RGBWW Lighting` model to 511.050

### PR DESCRIPTION
**Pull request by mrskycriper**
Thanks to @derkorte for pointing this model out as unsupported.

### CODE REVIEW

File: src/devices/iluminize.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ const definitions: Definition[] = [
         extend: [onOff()],
     },
     {
-        zigbeeModel: ['511.050'],
+        zigbeeModel: ['511.050', 'RGBWW Lighting'],
```
[the remark this answers is not in the document]
Comment (Nerivec) [reply src/devices/iluminize.ts:27]: https://github.com/Koenkk/zigbee-herdsman-converters/blob/cd9f978466c42106856e698eeb37bda3e5457534/src/devices/iluminize.ts#L130

https://github.com/Koenkk/zigbee-herdsman-converters/blob/cd9f978466c42106856e698eeb37bda3e5457534/src/devices/iluminize.ts#L34

Seems the converters for that manufacturer need sprucing up in general.

[the remark this answers is not in the document]
Comment (derkorte) [reply src/devices/iluminize.ts:27]: @Koenkk this seems wrong. The vendor is iluminize or Sunricher the Modell is the same: 511.050

[the remark this answers is not in the document]
Comment (mrskycriper) [reply src/devices/iluminize.ts]: Converted all of those to fingerprints too for good measure

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**